packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/03-quickstart.mdx

```diff
@@ -2,258 +2,145 @@
 title: Quickstart
 description: For when you're in a hurry and want to quickly get up and running
 ---
-
-:::warning
-Write React Native Specifics
-
-This documentation is for the beta version of the Stream Video SDK.
-The SDK is subject to change.
+import { TokenSnippet } from '../../../shared/_tokenSnippet.jsx';
+
+In this guide, we will cover the basics of making your first call using Stream Video.
+If you haven't already, we recommend starting with the __[introduction](../../)__ and __[installation](../installation)__ steps first, as this guide will build on the material covered in those sections.
+
+## Client setup & Calls
+
+Create an instance of `StreamVideoClient` that will establish WebSocket connection by connecting a user. Next you create a call object and join the call. We'll specify `create: true` to create the call if it doesn't exist.
+
+```tsx
+import {
+  StreamCall,
+  StreamVideo,
+  StreamVideoClient,
+  User,
+} from '@stream-io/video-react-native-sdk';
+import { useEffect, useState } from 'react';
+
+const apiKey = 'your-api-key';
+const userId = 'user-id';
+const token = 'authentication-token';
+const callId = 'my-call-id';
+const user: User = { id: userId };
+
+export default function App() {
+  const [client] = useState(
+    () => new StreamVideoClient({ apiKey, user, token }),
+  );
+  const [call] = useState(() => client.call('default',callId));
+
+  useEffect(() => {
+    call.join({ create: true });
+  }, [call]);
+
+  return (
+    <StreamVideo client={client}>
+      <StreamCall call={call}>{/* We will add the video UI in the next step */}</StreamCall>
+    </StreamVideo>
+  );
+}
+```
+
+`default` is a call type. There are 4 built-in [call types](../../core/configuring-call-types/). and you can also create your own.
```

Comment:
A bit more context after the code block will probably help. How about something like this:

```suggestion
When creating a call on the client, the string `default` is a call type. There are 4 built-in [call types](../../core/configuring-call-types/). and you can also create your own.
```

Reply:
Did you really mean "string" 🤣 (just kidding)